index.html

```diff
@@ -0,0 +1,176 @@
+        <h2>2022.10.15(THU) ~ 15(SAT)</h2>
+        <p>@ Espaço das Américas Brazil.</p>
+      </div>
+    </section>
+  </header>
+  <!-- Main Programs Section -->
+  <main>
+    <section class="main">
+      <h2 class="mobi-heading">Main Program</h2>
+
+      <hr class="mobi-hr">
+
+      <div class="main-grids">
+        <div class="box">
+          <img src="./images/lecture.png" alt="lecture pic">
+          <h3 class="box-heading">Lecture</h3>
+          <p class="box-text">
+            We will teach you everything that it takes to be a model in terms of fashion
+          </p>
+        </div>
+        <div class="box">
+          <img src="./images/face.png" alt="Vector">
+          <h3 class="box-heading">Men Fashion Exhibition</h3>
+          <p class="box-text">
+            Men Fashion has their own and unique styles.
+            Talks on dominance clothes that brings the calm and bold you
+          </p>
+        </div>
+        <div class="box">
+          <img src="./images/lady.png" alt="lady-vector">
+          <h3 class="box-heading">Women Fashion Exhibition</h3>
+          <p class="box-text">
+            Learn how to dress more elegant and feminine when it comes to fly your colors correctly
+          </p>
+        </div>
+        <div class="box">
+          <img src="./images/ropes.png" alt="ropes-vector">
+          <h3 class="box-heading">The Fashion Way</h3>
+          <p class="box-text">
+            Discussions that is on the trend about how to make everything better and harmonic
+          </p>
+        </div>
+        <div class="box">
+          <img src="./images/tie.png" alt="tie-vector">
+          <h3 class="box-heading">Tie The knot</h3>
+          <p class="box-text">
+            Time to communicate & do the networking! Get the chance to meet legendary fashion icons from all over
+            the world
+          </p>
+        </div>
+      </div>
+      <button class="join-button" type="button">join dc fashion show</button>
+      <a href="#" class="hide-on-mobile">See the whole program</a>
+    </section>
+
+    <!-- Featured Participants Section -->
+
+    <article class="article-container">
+      <div class="about-main-heding">
+        <h2 class="p-head">Featured Speakers's</h2>
+        <hr>
+      </div>
+      <section id="about" class="about">
+      </section>
+      <div class="button">
+        <button class="see-button" id="see-button" type="button">see more <i
+            class="fa-solid fa-chevron-down"></i></button>
+      </div>
+    </article>
+    
+  </main>
+
+  <!--Partenrs Section-->
+
+  <footer>
+    <div class="foot">
+      <div class="foot-heading">
+        <h1>Partner</h1>
+        <hr>
+      </div>
+      <div class="partners">
+        <div id="Stitch-up">STITCH UP</div>
+        <div id="Air-ups">Hope Fashion</div>
+        <div id="Mozzi-ups">Hot Make-up</div>
+        <div id="Air-up">Air UP</div>
+        <div id="Mozzi-up">Mozzi UP</div>
+      </div>
```

Comment:
- [ OPTIONAL] Great job so far 👏🏻 , however, you could improve this section by using semantic  HTML code what do you think to use a list tag  `ul > li`.  it's always a good practice to use `li` to represent an item in a list and a `ul ` to group them,  It could be something like this :point_down: 
    ```HTML 
          <ul>
                <li>STITCH UP</li>
                <li>>Hope Fashion<</li>
                <li> ....</li>
          </ul>
     ```

- I know you can do it 💪🏻 😄